iosApp/iosApp/Pages/StopDetails/StopDetailsFilteredDepartureDetails.swift

```diff
@@ -60,31 +70,54 @@ struct StopDetailsFilteredDepartureDetails: View {
                         .padding(.bottom, 6)
                         .dynamicTypeSize(...DynamicTypeSize.accessibility1)
 
-                        departureTiles(patternsByStop, view)
-                            .dynamicTypeSize(...DynamicTypeSize.accessibility3)
-                            .onAppear { if let id = tripFilter?.tripId { view.scrollTo(id) } }
+                        if !tiles.isEmpty {
```

Comment:
question(non-blocking):`tiles` and `noPredictionsStatus` mutually exclusive, right? If so, It might be worth making a single enum field to convey that, like `tilesOrStatus`